js/jsmodules.go

```diff
@@ -32,6 +33,7 @@ func getInternalJSModules() map[string]interface{} {
 		"k6/experimental/websockets": &expws.RootModule{},
 		"k6/experimental/timers":     timers.New(),
 		"k6/experimental/tracing":    tracing.New(),
+		"k6/experimental/browser":    browser.New(),
```

Comment:
To require that the browser module is explicitly enabled with an environment variable, you'd basically need a bogus `RootModule` implementation here on the k6 side. I think we can add a `browser` subfolder in [here](https://github.com/grafana/k6/tree/master/js/modules/k6/experimental) and add the module there. That module's `NewModuleInstance()` method can make the env var check and, if the env var is present, call xk6-browser's actual `RootModule.NewModuleInstance()`.

Also, a e2e test or two in https://github.com/grafana/k6/blob/master/cmd/tests/cmd_run_test.go for that check would be ideal :pray:

Reply:
I think the team would be happy to do that, which would mean that we're not going to create a generic permission like the one in the example [here](https://deno.land/manual@v1.15.2/getting_started/permissions)?

Reply:
Resolved in: 04a9daf157d477edd43c8790b7e43e74bb582c38 & 989169120543a6a4d43912864782f3e2a76b6889.